designsafe/apps/workspace/models/app_entries.py

```diff
@@ -176,6 +176,12 @@ class AppVariant(models.Model):
         blank=True,
     )
 
+    shortLabel = models.CharField(
+        help_text="The display name of this app in the Apps side navigation. If not defined, uses notes.shortLabel from app definition.",
```

Comment:
Can you make a second PR to main with this additional field and migration?

Reply:
Elected to keep this model kept to this PR, will rely on `notes.shortLabel` for prod apps in the interim